src-opam/dockerfile_distro.ml

```diff
@@ -51,9 +51,9 @@ let distro_status (d:t) : status = match d with
   | `Alpine `V3_9 -> `Active `Tier2
   | `Alpine `V3_10 -> `Active `Tier1
   | `Alpine `Latest -> `Alias (`Alpine `V3_10)
-  | `CentOS `V7 -> `Active `Tier2
-  | `CentOS `V6 -> `Deprecated
-  | `CentOS `Latest -> `Alias (`CentOS `V7)
+  | `CentOS `V8 -> `Active `Tier2
+  | `CentOS (`V7 | `V6) -> `Deprecated
```

Comment:
V7 should definitely not be deprecated - leave it as a Tier 2 for a while alongside CentOS 8